templates/vars/aks/sbox.json

```diff
@@ -19,6 +19,12 @@
     },
     "type": {
       "value": "VirtualMachineScaleSets"
+    },
+    "osDiskType": {
+      "value": "Ephemeral"
+    },
+    "vmSize": {
```

Comment:
should be removed from variable group too?

Reply:
yeah, agree.